momento-sdk/src/main/java/momento/sdk/config/StorageConfigurations.java

```diff
@@ -0,0 +1,123 @@
+package momento.sdk.config;
+
+import java.time.Duration;
+import momento.sdk.config.transport.GrpcConfiguration;
+import momento.sdk.config.transport.StaticTransportStrategy;
+import momento.sdk.config.transport.TransportStrategy;
+import momento.sdk.retry.DefaultRetryEligibilityStrategy;
+import momento.sdk.retry.FixedCountRetryStrategy;
+import momento.sdk.retry.RetryStrategy;
+
+/** Prebuilt {@link StorageConfiguration}s for different environments. */
+public class StorageConfigurations {
+
+  public static final int DEFAULT_MAX_RETRIES = 3;
+
+  /**
+   * Provides defaults suitable for a medium-to-high-latency dev environment. Permissive timeouts,
+   * retries, and relaxed latency and throughput targets.
+   */
+  public static class Laptop extends StorageConfiguration {
+
+    private Laptop(TransportStrategy transportStrategy, RetryStrategy retryStrategy) {
+      super(transportStrategy, retryStrategy);
+    }
+
+    /**
+     * Provides the latest recommended configuration for a dev environment.
+     *
+     * <p>This configuration may change in future releases to take advantage of improvements we
+     * identify for default configurations.
+     *
+     * @return the latest Laptop configuration
+     */
+    public static StorageConfiguration latest() {
+      return Laptop.v1();
+    }
+
+    /**
+     * Provides v1 configuration for a laptop environment. This configuration is guaranteed not to
+     * change in future releases of the Momento Java SDK.
+     *
+     * @return the v1 laptop configuration
+     */
+    public static StorageConfiguration v1() {
+      final TransportStrategy transportStrategy =
+          // TODO
+          new StaticTransportStrategy(new GrpcConfiguration(Duration.ofMillis(15000)));
+      final RetryStrategy retryStrategy =
+          new FixedCountRetryStrategy(DEFAULT_MAX_RETRIES, new DefaultRetryEligibilityStrategy());
```

Comment:
Is this eligibility strategy going to do anything here? It's checking for request types that only exist in the Cache APIs, right?

Reply:
We can add our own that defines types for the storage client.